src/components/organisms/Settings/Workspace/Asset/index.tsx

```diff
@@ -2,7 +2,7 @@ import React from "react";
 import { useIntl } from "react-intl";
 import useHooks from "./hooks";
 import SettingPage from "@reearth/components/organisms/Settings/SettingPage";
-import AssetSection from "@reearth/components/molecules/Settings/Workspace/Asset/AssetSection";
+import AssetSection from "@reearth/components/molecules/Settings/Workspace/AssetSection";
```

Comment:
Can you explain why AssetSection has been moved to Workspace?

AssetSection is used only inside Asset, so I think it should be in Asset directory.

Reply:
yes, that's true. I move this asset part from project, I guess I see the dir path look like this , which is ok.
![image](https://user-images.githubusercontent.com/3818046/122505427-4e473400-d037-11eb-9733-b5e3124dd36c.png)
any way, let me change it